parsl/providers/kubernetes/kube.py

```diff
@@ -34,6 +34,18 @@ class KubernetesProvider(ExecutionProvider, RepresentationMixin):
         Minimum number of blocks to maintain.
     max_blocks : int
         Maximum number of blocks to maintain.
+    max_cpu : float
+        CPU limits of the blocks (pods).
+        In cpu units, check kubernetes docs for more details. Default is 2.
+    max_mem : str
+        Memory limits of the blocks (pods).
+        In Mi or Gi, check kubernetes docs for more details. Default is 500Mi.
```

Comment:
For existing kubernetes users, it would be useful to make it clear that there fields correspond to kubernetes side `limit` and `request`. That is obvious looking at this PR but maybe won't be later.

Reply:
Added this to docstrings.